backend/btrixcloud/colls.py

```diff
@@ -1175,6 +1209,24 @@ async def download_public_collection(
 
         return await colls.download_collection(coll.id, org)
 
+    @app.get(
+        "/public/orgs/{org_slug}/collections/{coll_slug}/thumbnail",
```

Comment:
We're always saving thumbnails as jpgs, right? Should this be `.../thumbnail.jpg`?

Reply:
It looks like we're guessing and storing the mime https://github.com/webrecorder/browsertrix/blob/main/backend/btrixcloud/models.py#L1205
and potentially with user uploads, may have non-JPEGs also, eg. PNGs at least? Probably better to leave as is to be flexible

Reply:
Okay, sounds good!